qiskit_ibm_runtime/sampler.py

```diff
@@ -257,12 +259,30 @@ def _call(
         combined = self.options._merge_options(run_options)
         inputs.update(Options._get_program_inputs(combined))
 
-        return self._session.run(
+        raw_result = self._session.run(
             program_id=self._PROGRAM_ID,
             inputs=inputs,
             options=Options._get_runtime_options(combined),
             result_decoder=SamplerResultDecoder,
         ).result()
+        quasi_dists = []
+        for quasi, meta in zip(raw_result["quasi_dists"], raw_result["metadata"]):
+            shots = meta.get("shots", float("inf"))
+            overhead = meta.get("readout_mitigation_overhead", 1.0)
```

Comment:
This differs from my suggestion if `meta['shots'] == None` or `meta['readout_mitigation_overhead'] == None`. Is that possible? If so then the code below would raise an error.

Reply:
These metadata are not designed to be set `None`. So, I think it's no problem.